app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,18 @@
 .downvote {
   color: red;
 }
+
+#foto, #foto img {
+    background: #1ABC9C;
+    border-radius:50%;
+}
+#foto {
+    border-radius: 50%;
+    border: 15px solid #34495E;
```

Comment:
Color literals like `#34495E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#34495E` should be written as `#34495e`<br>Line should be indented 2 spaces, but was indented 4 spaces